x/slashing/params.go

```diff
@@ -12,15 +12,19 @@ const (
 	DefaultParamspace = "slashing"
 )
 
+// The Double Sign Jail period ends at Max Time (tombstone)
```

Comment:
Can we also actually add the data here in the comment?

Reply:
What do you mean?

Reply:
Like the actual RFC3339 date.

Reply:
Oh!  Date, not data!

Reply:
derp! Sorry :-p